README.md

```diff
@@ -2,6 +2,9 @@
 
 [![GoDoc](https://godoc.org/github.com/lionelbarrow/braintree-go?status.svg)](http://godoc.org/github.com/lionelbarrow/braintree-go) [![Build Status](https://travis-ci.org/lionelbarrow/braintree-go.png?branch=master)](https://travis-ci.org/lionelbarrow/braintree-go)
 
+> [!NOTE]
```

Comment:
See the link to preview the display style of the note - [add-fork-disclaimer#readme](https://github.com/processout/braintree-go/tree/docs/POINT-1120/add-fork-disclaimer#readme)